sdk/core/azure-core-test/src/main/java/com/azure/core/test/http/TestProxyTestServer.java

```diff
@@ -47,7 +60,13 @@ public TestProxyTestServer() {
                 .get("/fr/path/2",
                     (req, res) -> res.status(HttpResponseStatus.OK)
                         .addHeader("Content-Type", "application/json")
-                        .sendString(Mono.just(TEST_XML_RESPONSE_BODY))))
+                        .sendString(Mono.just(TEST_XML_RESPONSE_BODY)))
+                .get("/fr/path/3", (req, res) -> {
```

Comment:
I know this is just testing but I think this is a great test we should leave in, so let's make the path reflect that. `/getRedirect` or something.